spec/features/reply_from_councillor_spec.rb

```diff
@@ -1,10 +1,8 @@
 require "spec_helper"
 
 feature "Councillor replies to a message sent to them" do
-  given (:councillor) { create(:councillor, name: "Louise Councillor", email: "louise@council.nsw.gov.au") }
-  given (:application) { create(:application, id: 8, address: "24 Bruce Road Glenbrook", description: "A lovely house") }
-  given (:comment) do
   let(:councillor) { create(:councillor, name: "Louise Councillor", email: "louise@council.nsw.gov.au", popolo_id: "marrickville_council/chris_woods") }
+  let(:application) { create(:application, id: 8, address: "24 Bruce Road Glenbrook", description: "A lovely house") }
```

Comment:
@equivalentidea.  Got it. Hadn't used that syntax before so thought it must have been old. :)
Reverted back now to use `given` and `background`